app/views/account_requests/new.html.erb

```diff
@@ -1,8 +1,13 @@
 <% unless Rails.env.staging? %>
-  <div class="card" style='width: 30rem'>
-    <div class="card-body">
-      <div class="card-title mb-10">
-        <h3>Account Request Form</h3>
+  <div class="bg-white" style='width: 35rem'>
+    <div class="p-3">
+      <div class="mb-10">
+        <h1 class='text-3xl text-center'>Essentials Bank Account Request Form</h1>
+        <p class='p-2 bg-yellow-200'>
+          Fill out the details below about your bank to request a free account to use Human Essentials for your organization's essentials bank.
+          <br><br>
+          <span class='text-bold'>If you are a partner please go <%= link_to 'here', new_partner_user_session_path %> to login.</span
```

Comment:
On verbiage: Is 'partner' a known term outside of the diaper bank users? It has a specific meaning to us that might not be immediately obvious to someone we consider a 'partner'. To put it in more explicit terms, maybe something like
```suggestion
          <span class='text-bold'>Do you <span class='text-italic'>partner</span> with diaper and/or period supply banks? If so please go to the <%= link_to 'partner app', new_partner_user_session_path %> to login.</span
```

Reply:
Hmm. Might also make sense to have verbiage that says to contact the diaper/period supplies bank if you don't yet have an account. Right? I'm a user who ended up on the request-an-account form for diaper banks. If I'm here, I might be trying to sign up, but I won't be able to do that when I go to the Partner login page.

Reply:
@scooter-dangle great suggestion! I think adding that would be useful here. Maybe also on the login page too? What do you think about adding that here and in the login?

Reply:
> What do you think about adding that here and in the login?

Yeah, I think that makes sense.

> maybe something like 'Requestor' or something makes a bit more sense?

Whatever name, I think it's probably useful to make the connection between that label and the more concrete, plain-language description of what their relationship is with the diaper bank. (Specifically at entrypoints to the application like this—once they're in, it doesn't seem super important. Just want to make sure they understand what they're asking for here so we cut down on the mistaken account requests.)